node.go

```diff
@@ -86,12 +86,16 @@ func (n *Node) LookupByString(s string) (ipld.Node, error) {
 	return n.lookupValue(&n.Hamt, n.bitWidth(), 0, hk, key)
 }
 
-func (*Node) LookupByNode(ipld.Node) (ipld.Node, error) {
-	panic("TODO")
+func (n *Node) LookupByNode(key ipld.Node) (ipld.Node, error) {
+	ks, err := key.AsString()
```

Comment:
Will keys always be a string?
Wonder if it makes sense to also handle cases where the type is Bytes?

Reply:
Ah, you're right, I assumed same interface as the Filecoin HAMT which always passes keys as strings but we could support bytes too for more flexibility.
Also do you think I could get added to the IPLD org? Would love to help maintain and contribute as much as I can.

Reply:
> do you think I could get added to the IPLD org?

that part is self-serve now. Propose adding yourself with a PR modifying https://github.com/ipld/github-mgmt/blob/master/github/ipld.yml

Reply:
Thanks!